Cargo.lock

```diff
@@ -937,7 +941,7 @@ checksum = "3f9eec918d3f24069decb9af1554cad7c880e2da24a9afd88aca000531ab82c1"
 
 [[package]]
 name = "forc"
-version = "0.3.3"
+version = "0.4.0"
```

Comment:
Ahh, could you make a separate PR to bump the version of these in the lockfile? Looks like #760 forgot the bump the lockfile.